src/pyrenew/process/differencedprocess.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+# numpydoc ignore=GL08
+
+from __future__ import annotations
+
+import jax.numpy as jnp
+from jax.typing import ArrayLike
+from numpyro.contrib.control_flow import scan
+
+from pyrenew.metaclass import RandomVariable, SampledValue
+
+
+class DifferencedProcess(RandomVariable):
+    """
+    Class for differenced stochastic process X(t),
+    constructed by placing a fundamental stochastic
+    process on the :math:`n^{th}` differences
+    (rates of change). See
+    https://otexts.com/fpp2/stationarity.html
```

Comment:
```suggestion
    https://otexts.com/fpp3/stationarity.html#differencing
```

or to be more specific

Reply:
I think it makes more sense to start from the top, since the actual chapter title is "Stationarity and differencing"